document/src/main/java/com.google.android.fhir.document/scan/ScannerUtils.kt

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright 2023-2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.fhir.document.scan
+
+import android.content.Context
+import androidx.camera.core.ImageAnalysis
+import androidx.lifecycle.LifecycleOwner
+import com.google.mlkit.vision.barcode.Barcode
+import com.google.mlkit.vision.barcode.BarcodeScannerOptions
+import com.google.mlkit.vision.barcode.BarcodeScanning
+
+class ScannerUtils(
```

Comment:
It is good practice to use composition, and pass in the classes as parameters as opposed to instantiating them in the class itself. this will make testing easier as you can mock the passed classes.

```suggestion
class Scanner(
  private val cameraManager: CameraManager,
  private val  barcodeDetectorManager: BarcodeDetectorManager,
)
```